python/use_python.py

```diff
@@ -10,7 +10,7 @@
 from py_astro_accelerate import *
 
 # Open filterbank file for reading metadata and signal data
-sigproc_input = aa_py_sigproc_input("<input_some_filterbank_data.fil>")
+sigproc_input = aa_py_sigproc_input("/home/novotny/filterbank/ska-mid-b2-small.fil")
```

Comment:
Leave old string for fil file

Reply:
Done, also added a comment to the user to add the filterbank file